build.gradle

```diff
@@ -9,9 +9,15 @@ buildscript {
 }
 
 plugins {
+  id "com.gradle.build-scan" version "1.16"
```

Comment:
Can you use the 2.0+ version? Requires Gradle 5.0.
See: https://github.com/jaredsburrows/android-gif-example/blob/master/gradle/scan.gradle.kts

Reply:
The current build sets `4.10.1` in the wrapper, that's why the chosen version of `build-scan` is `1.16'. If the build migrates to `5.x` then yes, by all means let's upgrade the `build-scan` plugin :smile:

Reply:
There is no reason not to update, I don't think there should be a compatibility issue to do so. Can you give it a quick try?

Reply:
There's one potential issue though. Gradle `5.x` includes a more recent version of Groovy and that may cause a compatibility issue. If upgraded then the next version of this plugin would force consumers to use Gradle `5.x`. Is that desirable?

Reply:
That's a great point. I'm fine either way since the older versions are available, but would lean towards your mindset of holding back a bit to allow 5.x to spread into less actively maintained projects. What do you think @jaredsburrows?